frontend/src/app/components/label-explorer/label-explorer.component.html

```diff
@@ -30,20 +30,21 @@
                         Slice #{{ label.sliceIndex }}
                     </p>
                     <p class="label-item__subheader" matLine>
-                        Selection #{{ label.selectionId }}
+                        Selection #{{ label.getId() }}
```

Comment:
Maybe it would be better to have:
```suggestion
                        Selection #{{ label.getSelectionId() }}
```

Reply:
Maybe this way, I renamed variable `label` to `selection` and now we have `Selection #{{ selection.getId() }}`